src/reanimated2/core.ts

```diff
@@ -28,7 +28,7 @@ import type {
 import { SensorContainer } from './SensorContainer';
 
 export { startMapper, stopMapper } from './mappers';
-export { runOnJS, runOnUI } from './threads';
+export { runOnJS, runOnUI, executeOnUIRuntimeSync } from './threads';
```

Comment:
We will need to add this function to our docs 🤯

Reply:
that is true 🫣 but maybe save it for another PR